apps/mobile/src/queries/balance/use-stx-balances.query.ts

```diff
@@ -0,0 +1,51 @@
+import { useCallback } from 'react';
+
+import { useMempoolTxsBalance } from '@/queries/stacks/use-mempool-txs-balance';
+import { useQueries } from '@tanstack/react-query';
+
+import {
+  AddressBalanceResponse,
+  createGetStacksAccountBalanceQueryOptions,
+  createStxCryptoAssetBalance,
+  createStxMoney,
+  useCurrentNetworkState,
+  useStacksClient,
+} from '@leather.io/query';
+import { createMoney, sumMoney } from '@leather.io/utils';
+
+export function useStxBalancesQueries(addresses: string[]) {
+  const client = useStacksClient();
+  const network = useCurrentNetworkState();
+
+  const { inboundBalance, outboundBalance } = useMempoolTxsBalance(addresses);
```

Comment:
I think that maybe I should keep going and move `useMempoolTxsBalance` into this `useQueries` also